GoogleForCreators/web-stories-wp
# Single blob shape + fix blob alignment

## Summary

Temporarily removes the 8 different blob shapes and replaces them with a single shape in preparation for #686 

## Relevant Technical Choices

Added a migration script that switches all blobs to the single blob object.

## User-facing changes

❗❗ **This alters the appearance of stories that used the old blobs in two ways: Stories that used `blob-1` will now see a bigger blob since the path was changed to fill empty space and stories that used other blobs will be replaced by `blob-1`** 

## Testing Instructions

1. Drag a blob element into the canvas
2. Notice how the outline, drop target and frame align to the blob's shape (no longer showing empty space around it)
---



Fixes #1896
